packages/workbox-webpack-plugin/src/lib/generate-webpack-sw.js

```diff
@@ -0,0 +1,29 @@
+const {readFile} = require('./utils/read-file');
+
+const importScripts = ({
```

Comment:
Not: I'd rename this function. importScripts is way to generic / confusing given the context. `wrapAsImport` or something in that direction would be helpful.